docs/04-operation-guides/troubleshooting/smsh-01-503-no-access.md

```diff
@@ -38,15 +43,15 @@ If this solution doesn't work, you need to change the image of the Istio Ingress
     ```bash
     kubectl logs -n istio-system -l app=istio-ingressgateway -c ingress-sds
     ```
-   
+
 5. In case of certificate-related issues, make sure `kyma-gateway-certs` and `app-connector-certs` Secrets are available in the `istio-system` Namespace and that they contain proper data. Run:
 
     ```bash
     kubectl get secrets -n istio-system kyma-gateway-certs -oyaml
     kubectl get secrets -n istio-system app-connector-certs -oyaml
     ```
 
-6. To regenerate a corrupted certificate, follow [this tutorial](components/security/#tutorials-update-tls-certificate). If you are running Kyma provisioned through Gardener, follow [this tutorial](components/security/#troubleshooting-issues-with-certificates-on-gardener) instead.
+<!-- Update step 6 once the long-lasting certificate is implemented. Probably, only the details about Gardener will be needed. -->
+6. To regenerate a corrupted certificate, follow [this tutorial](../../../03-tutorials/sec-01-tls-certificates-security.md). If you are running Kyma provisioned through Gardener, follow [this tutorial](./sec-02-certificates-gardener.md) instead.
```

Comment:
Both links do not work 🤔

Reply:
- The first link should work after merging the PR (the PR restructuring the Security docs was not merged when I was creating a new branch for the Service Mesh restructuring. In the new version of Security docs the tutorial is here -> https://deploy-preview-11747--kyma-project-docs-preview.netlify.app/docs/kyma/preview/03-tutorials/sec-01-tls-certificates-security/).
- Also the location of the second document was a little bit changed to `04-operation-guides/troubleshooting/sec-01-certificates-gardener/`